### ISSUE TITLE
Test "test_input_output" from Adjoint Application is failing when run from the main test script

### ISSUE THREAD

**Issue by jcotela** [KratosMultiphysics/Kratos#271]
If I try to run the main python test script `kratos/kratos/python_scripts/run_test.py` I get a failure for the Adjoint Application with the following message:

```
ERROR: test_Execution (test_input_output.TestCase)
----------------------------------------------------------------------
  Traceback (most recent call last):
  File "/home/jcotela/kratos/applications/AdjointFluidApplication/tests/test_input_output.py", line 42, in tearDown
    if "io_test.time" in os.listdir("./test_input_output"):
FileNotFoundError: [Errno 2] No such file or directory: './test_input_output'
```

The tests run OK if I just run the main python file for the Adjoint Application, so I guess this is due to the use of relative paths in the test files (since runtime paths are different depending on the script I launch). If so, the solution would be to define an auxiliary function such as `GetFilePath` in `kratos/kratos/python_scripts/test_gid_io.py`.

@msandre Is it OK for you if I go ahead and do this?

**Comment by msandre** [comment]
Hi Jordi,

I can't test this until I am back in the office, but the code in test_input_output.py should take care to cd to the test directory so the relative path works. Obviously there's something wrong with it.

**Comment by msandre** [comment]
I had a missing execution scope in the test. I added a fix for the error in pr #274. It should work now without modifications to the core.

**Comment by jcotela** [comment]
Closing, solved by #274
